doc/modules/mod_mam.md

```diff
@@ -158,3 +157,9 @@ You can change the default settings using extra parameters:
               ]}
        ]}.
 ```
+
+### Backend metrics
+
+* `[global, backends, mod_mam, lookup]` - A time of archive lookup operation.
+* `[global, backends, mod_mam, archive]` - A time of saving one message in the archive.
```

Comment:
time it takes to save a message in the archive

or, if you mean a point in time rather than duration:

time of saving a message in the archive